projects/ngrx.io/content/guide/migration/v9.md

```diff
@@ -0,0 +1,146 @@
+# V9 Update Guide
+
+## Angular CLI update
+
+NgRx supports using the Angular CLI `ng update` command to update your dependencies. Migration schematics are run to make the upgrade smoother. These schematics will fix breaking changes.
+
+To update your packages to the latest released version, run the command below.
+
+```sh
+ng update @ngrx/store
+```
+
+## Dependencies
+
+Version 9 has the minimum version requirements:
+
+- Angular version 9.x
+- Angular CLI version 9.x
+- TypeScript version 3.7.x
+- RxJS version 6.5.x
+
+<div class="alert is-helpful">
+
+NgRx version 9 can be used with earlier versions of Angular. TK
```

Comment:
This should be removed. Even though this is true, we don't want to officially support doing this.